include/open62541/plugin/certificategroup.h

```diff
@@ -40,6 +40,31 @@ struct UA_CertificateGroup {
     void (*clear)(UA_CertificateGroup *certGroup);
 };
 
+struct UA_Transaction;
+typedef struct UA_Transaction UA_Transaction;
+
+struct UA_Transaction {
+    UA_NodeId sessionId;
+
+    size_t certGroupSize;
+    UA_CertificateGroup *certGroups;
+
+    /* The entries in the arrays are related to each other at the same index. */
+    size_t serverCertSize;
+    UA_ByteString *certificates;
+    UA_ByteString *privateKeys;
+    UA_NodeId *certificateGroups;
+    UA_NodeId *certificateTypes;
```

Comment:
Does it make sense to combine the items (certificate, key, group and type) into a new structure to make this relation clear?